dhis-2/pom.xml

```diff
@@ -833,7 +830,6 @@
                 <RestrictImports>
                   <reason>Use javax.annotation.Nonnull and javax.annotation.CheckForNull instead</reason>
                   <bannedImports>
-                    <bannedImport>javax.validation.constraints.NotNull</bannedImport>
```

Comment:
Are you removing it since we removed the dependency that includes this class?
Just checking since I think this was added to guide us in https://github.com/dhis2/wow-backend/blob/master/guides/preferred_patterns.md#use-nonnull-in-apis

Reply:
I think we can remove it or keep it just to be safe. Both should work.

Reply:
Removed because the dependency is not there anymore